pkg/controller/nodelifecycle/scheduler/rate_limited_queue.go

```diff
@@ -194,6 +194,15 @@ func (q *UniqueQueue) Clear() {
 	}
 }
 
+// SetRemove remove value from the set if value existed
+func (q *UniqueQueue) SetRemove(value string) {
```

Comment:
prefer to rename to `Remove`; the other parts are LGTM :)

Reply:
OK，i will correct it

Reply:
There has been a `func (q *UniqueQueue) Remove (value string) bool `method, which used to remove from queue.